packages/server/jiraImagesHandler.ts

```diff
@@ -1,32 +1,57 @@
 import {HttpRequest, HttpResponse} from 'uWebSockets.js'
+import fs from 'fs'
 import mime from 'mime-types'
 import sleep from '../client/utils/sleep'
 import uWSAsyncHandler from './graphql/uWSAsyncHandler'
-import {NO_IMAGE_BUFFER} from './utils/atlassian/jiraImages'
 import getRedis from './utils/getRedis'
+import jiraPlaceholder from '../../static/images/jira/placeholder.png'
+
+const jiraPlaceholderBuffer = fs.readFileSync(jiraPlaceholder)
```

Comment:
I'm not sure what's the best way of reading the file in a node so @mattkrick I'd appreciate if you could take a look here! Thanks!

Reply:
let's do a lazy + async read.
we'll make it async so it doesn't block the event loop
we'll make it lazy so the server doesn't have to read the file every time it starts up

```ts
import {promises as fsp} from 'fs'
let jiraPlaceholderBuffer: Buffer | undefined

const servePlaceholderImage = async (res: HttpResponse) => {
  if (!jiraPlaceholderBuffer) {
    jiraPlaceholderBuffer = await fsp.readFile(jiraPlaceholder)
  }
  res.cork(() => {
    res
      .writeStatus('200')
      .writeHeader('Content-Type', 'image/png')
      .end(jiraPlaceholderBuffer)
  })
}
```

Reply:
Thank you, all noted!